taskboot/docker.py

```diff
@@ -174,6 +178,18 @@ def push_archive(self, path):
             self.run(cmd)
             logger.info('Push successfull')
 
+    def docker_id_archive(self, path):
+        '''Get docker image ID
+
+        Docker image ID corresponds to the sha256 hash of the config file
+        '''
+        # Imported from https://github.com/mozilla/release-services/blob/master/lib/please_cli/please_cli/utils.py#L186
+        tar = tarfile.open(path)
+        manifest = json.load(tar.extractfile('manifest.json'))
```

Comment:
Because it's not reading the file-like object, it's directly read and decoded by the JSON Decoder. Given the size of the manifest.json, it shouldn't matter